installer.sh

```diff
@@ -56,8 +56,8 @@ while [ $# -gt 0 ]; do
       printf "*  --travis_enterprise_host=\"demo.enterprise.travis-ci.com\"  *\\n"
       printf "*  --travis_enterprise_security_token=\"token123\"             *\\n"
       printf "*  --travis_enterprise_build_endpoint=\"build-api\"            *\\n"
-      printf "*  --travis_queue_name=\"builds.linux\"                        *\\n"
-      printf "*  --travis_legacy_build_images=true                         *\\n"
+      printf "*  --travis_queue_name=\"builds.trusty\"                       *\\n"
```

Comment:
I believe we left `builds.linux` for the case of precise workers, especially given how the default on 2.1.x is precise. 

Instead of renaming this queue, I think we should add another `builds.xenial` one? I might be wrong though

Reply:
Ahh, got it. 

This line here only configures the queue this particular machine will listen to by default. Also, the user is able to override this by passing `--travis_queue_name="builds.linux"`.